src/execution/physical_operator.cpp

```diff
@@ -10,7 +10,7 @@
 namespace duckdb {
 
 string PhysicalOperator::GetName() const {
-	return PhysicalOperatorToString(type);
+	return PhysicalOperatorToString(type) + "_" + std::to_string(id);
 }
```

Comment:
Ahh I see - so we're just adding `_{id}` at the top level here. Does this affect anything else in DuckDB? 🤔 or has this historically just been informational?

Reply:
hm not sure. so far didn't run into any issue! this is just helpful for now when writing the lineage queries.